Java/formatter/intellij-java-google-style-pre-14.1.xml

```diff
@@ -12,8 +12,8 @@
       <option name="USE_RELATIVE_INDENTS" value="false" />
     </value>
   </option>
-  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="4" />
```

Comment:
I mean that it won't change much if we leave it to IntelliJ. People (me included) press the `Cmd+Alt+O` and don't pay attention. There is no way to ensure that the way imports are defined is meaningful. Anyway, LGTM, no-one will be hurt by that.

👍